anomalib/config/config.py

```diff
@@ -205,6 +205,18 @@ def get_configurable_parameters(
     # Project Configs
     project_path = Path(config.project.path) / config.model.name / config.dataset.name
 
+    if config.dataset.format == "folder":
+        if "mask" in config.dataset.keys():
```

Comment:
`refurbish` tool does not like keys. We could also address it once we add `refurbish` to pre-commit, but it would be good practice to get use to it

```suggestion
        if "mask" in config.dataset:
```